tests/bugs/gh-3087-multi-entry-point.slang

```diff
@@ -5,8 +5,8 @@
 
 // we should only have 1 'BuiltIn InstanceIndex' since the `Output` and `Input` semantic
 // for `InstanceIndex` should be converted to a non-builtin
-// CHECK-DAG: BuiltIn InstanceIndex
-// CHECK-NOT: BuiltIn InstanceIndex
+// CHECK: OpDecorate %gl_InstanceIndex BuiltIn InstanceIndex
+// CHECK-NOT: OpDecorate %gl_InstanceIndex BuiltIn InstanceIndex
 
 // We require 1 `Flat` for the fragment input `uint`
 // SPIRV: Flat
```

Comment:
Note: problem with the test, not your PR

I think `SPIRV:` is meant to be a `CHECK:` operation
if so, we should be using `CHECK-DAG` since `OpDecorate` may be emitting in any order
```suggestion
// CHECK-DAG: OpDecorate %gl_InstanceIndex BuiltIn InstanceIndex
// CHECK-NOT: OpDecorate %gl_InstanceIndex BuiltIn InstanceIndex

// We require 1 `Flat` for the fragment input `uint`
// CHECK-DAG: Flat
```
```suggestion
// CHECK-DAG: OpDecorate %gl_InstanceIndex BuiltIn InstanceIndex
// CHECK-NOT: OpDecorate %gl_InstanceIndex BuiltIn InstanceIndex

// We require 1 `Flat` for the fragment input `uint`
// SPIRV-DAG: Flat
```